utils/utils.go

```diff
@@ -1,11 +1,13 @@
+// Package Utils provides basic utilities for the errand server.
+//nolint:golint,stylecheck // TODO
 package utils
 
 import (
 	"time"
 )
 
 func GetTimestamp() int64 {
-	return (time.Now().UnixNano() / 1000000)
+	return (time.Now().UnixNano() / 1_000_000)
```

Comment:
didn't you just write something to wrap these kind of things?

Reply:
Sure, I'll use that.

Reply:
Can't.  This is a public repo.